macos/mac_os_resource_snd.ksy

```diff
@@ -0,0 +1,280 @@
+        -orig-id: instrumentChunks
+        type: u4
+        doc: pointer to instrument info
+      - id: aes_recording
+        -orig-id: AESRecording
+        type: u4
+        doc: pointer to audio info
+      - id: sample_size
+        -orig-id: sampleSize
+        type: u2
+        doc: number of bits per sample
+      - id: reserved
+        -orig-id: futureUse1, futureUse2, futureUse3, futureUse4
+        size: 14
+        doc: reserved
+      - id: sample_area
+        -orig-id: sampleArea
+        size: num_frames * num_channels * sample_size / 8
+        doc: sampled-sound data
+  compressed_sound_header:
+    seq:
+      - id: sample_ptr
+        -orig-id: samplePtr
+        type: u4
+        doc: pointer to samples (or 0 if samples follow data structure)
+      - id: num_channels
+        -orig-id: numChannels
+        type: u4
+        doc: number of channels in sample
+      - id: sample_rate
+        -orig-id: sampleRate
+        type: unsigned_fixed_point
+        doc: The rate at which the sample was originally recorded.
+      - id: loop_start
+        -orig-id: loopStart
+        type: u4
+        doc: loop point beginning
+      - id: loop_end
+        -orig-id: loopEnd
+        type: u4
+        doc: loop point ending
+      - id: encode
+        type: u1
+        enum: sound_header_type
+        doc: sample's encoding option
+      - id: base_frequency
+        type: u1
+        doc: base frequency of sample
+        -orig-id: baseFrequency
+      - id: num_frames
+        type: u4
+      - id: aiff_sample_rate
+        size: 10
+        doc: rate of original sample (Extended80)
+      - id: marker_chunk
+        -orig-id: markerChunk
+        type: u4
+        doc: reserved
+      - id: format
+        size: 4
+        type: str
+        encoding: ASCII
+        doc: data format type
+      - id: reserved
+        -orig-id: futureUse2
+        size: 4
+        doc: reserved
+      - id: state_vars_ptr
+        -orig-id: stateVars
+        type: u4
+        doc: pointer to StateBlock
+      - id: left_over_samples_ptr
+        -orig-id: leftOverSamples
+        type: u4
+        doc: pointer to LeftOverBlock
+      - id: compression_id
+        -orig-id: compressionID
+        type: u2
+        doc: ID of compression algorithm
+      - id: packet_size
+        -orig-id: packetSize
+        type: u2
+        doc: number of bits per packet
+      - id: synthesizer_id
+        -orig-id: snthID
+        type: u2
+        doc: resource ID of Sound Manager Synthesizer (snth)
+      - id: sample_size
+        -orig-id: sampleSize
+        type: u2
+        doc: number of bits per sample
+      - id: sample_area
+        -orig-id: sampleArea
+        size-eos: true
+        doc: compressed sound data
+  unsigned_fixed_point:
+    seq:
+      - id: integer_part
+        type: u2
+        doc: Integer Part of a fixed point value
+      - id: fraction_part
+        type: u2
+        doc: Fraction Part of a fixed point value
```

Comment:
I'd say that these `doc`s are just trivial transcriptions of `id`s, and they bring no new information. As such, they [should be removed](https://doc.kaitai.io/ksy_style_guide.html#seq-attr):

> Attributes SHOULD have at least `id` and `doc` — however, see below for notes about omitting `id`, and **`doc` SHOULD NOT be included if it’s trivial** (i.e. if it is a copy of `id`, and there is really nothing more to say about that attribute).

```suggestion
      - id: fraction_part
        type: u2
```